packages/esm-service-queues-app/src/hooks/useConcept.ts

```diff
@@ -0,0 +1,13 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr/immutable';
+import { type Concept } from '../types';
+
+export function useConcept(uuid: string) {
+  const apiUrl = `${restBaseUrl}/concept/${uuid}`;
```

Comment:
Basically allow us to _not_ make a request if the UUID isn't yet available.

```suggestion
  const apiUrl = uuid ? `${restBaseUrl}/concept/${uuid}` : undefined;
```